docs/dashboard-setup/whitelisting.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Client ID and Whitelisting references
+image: "images/docs-meta-cards/documentation-card.png"
+displayed_sidebar: docs
+description: "Client ID and Whitelisting references | Documentation - Web3Auth"
+---
+
+import DashboardImage from "@site/static/dashboard/project_plug_n_play.png";
+
+### Whitelisting URLs using the Developer Dashboard
+
+The Whitelist URL is the address you wish to authorize for your project. When implementing Web3Auth, it is important to whitelist the specific URLs
+from which you intend to make calls for user authentication. By doing so, you can ensure that these activities are conducted securely and efficiently.
+Remember to whitelist all relevant URLs to guarantee seamless integration with Web3Auth.
+
+Type in the URL you wish to whitelist in the text box and click on the `Add` button. The URL will be added to the list of whitelisted URLs. You can
+add as many URLs as you want.
+
+<div style={{ flexBasis: "300px", flexGrow: "1", textAlign: "center" }}>
+  <img src={DashboardImage} style={{ alignSelf: "center", maxWidth: "100%" }} alt="Dashboard Whitelist" />
+</div>
+
+### Whitelisting URLs using the OpenloginAdapter
```

Comment:
This should be the third point, not second, move this down

Rename this to Manual Whitelisting for Advanced Cases

Add a warning there that this is a risky process since client secret should be confidential. Please do this in a highly secured environment.